changelog/unreleased/issue-1756

```diff
@@ -0,0 +1,11 @@
+Bugfix: Mark repository files as readonly when using the local backend
+
+Files stored in a local repository are marked as readonly on non-Windows
```

Comment:
Might perhaps be good to write "marked as read-only in the filesystem", so it's clear where/how it's being marked as read-only.

Reply:
Is changed.